README.md

```diff
@@ -6,7 +6,7 @@ Imagefit allows you to render an image in a template and specify its dimensions.
 It preserves the original image file.
 
 It is compatible with various sources of images such as django-filebrowser's
-FileBrowseField, user uploaded images, static images, …
+FileBrowseField, user uploaded images, static images, ...
```

Comment:
I'd rather have an ellipsis character than 3 dots to emulate it. Did you meet any trouble with that?